dotnet/Gherkin/CucumberMessages/Types/PickleStep.cs

```diff
@@ -1,28 +1,31 @@
+using System.Diagnostics.CodeAnalysis;
 using System.Runtime.Serialization;
 
 namespace Gherkin.CucumberMessages.Types;
 
 public class PickleStep
 {
     [DataMember(Name = "argument")]
-    public PickleStepArgument Argument { get; set; }
+    public PickleStepArgument? Argument { get; set; }
 
     [DataMember(Name = "astNodeIds")]
-    public IReadOnlyCollection<string> AstNodeIds { get; set; }
+    public required IReadOnlyCollection<string> AstNodeIds { get; set; }
 
     [DataMember(Name = "id")]
-    public string Id { get; set; }
+    public required string Id { get; set; }
 
     [DataMember(Name = "text")]
-    public string Text { get; set; }
+    public required string Text { get; set; }
 
     [DataMember(Name = "type")]
-    public StepKeywordType Type { get; set; }
+    public required StepKeywordType Type { get; set; }

```

Comment:
According to the schema for Cucumber Messages, the Type property of a PickleStep is not required.

Reply:
thank you for feedback, fixed